pkg/api/routes.go

```diff
@@ -656,6 +656,10 @@ func (rh *RouteHandler) UpdateManifest(response http.ResponseWriter, request *ht
 		}
 	}
 
+	if subjectDigest.String() != "" {
+		response.Header().Set(constants.SubjectDigestKey, subjectDigest.String())
```

Comment:
Where is the "OCI-Subject-Processed" header set?

Reply:
In the latest version of the [distribution spec](https://github.com/opencontainers/distribution-spec/blob/main/spec.md#pushing-manifests-with-subject) there is no "OCI-Subject-Processed" header. It was changed to "OCI-Subject".